tests/filecheck/transforms/convert_memref_stream_to_loops.mlir

```diff
@@ -374,9 +370,61 @@ func.func @interleaved_no_init(%A : memref<3x5xf64>, %B : memref<5x8xf64>, %C :
+// CHECK-NEXT:          %{{.*}}, %{{.*}}, %{{.*}}, %{{.*}} = scf.for %{{.*}} = %{{.*}} to %{{.*}} step %{{.*}} iter_args(%c0 = %{{.*}}, %c1 = %{{.*}}, %c2 = %{{.*}}, %c3 = %{{.*}}) -> (f64, f64, f64, f64) {
+// CHECK-NEXT:            %a0 = memref.load %A[%{{.*}}, %{{.*}}] : memref<3x5xf64>
+// CHECK-NEXT:            %a1 = memref.load %A[%{{.*}}, %{{.*}}] : memref<3x5xf64>
+// CHECK-NEXT:            %a2 = memref.load %A[%{{.*}}, %{{.*}}] : memref<3x5xf64>
+// CHECK-NEXT:            %a3 = memref.load %A[%{{.*}}, %{{.*}}] : memref<3x5xf64>
+// CHECK-NEXT:            %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:            %b0 = memref.load %B[%{{.*}}, %{{.*}}] : memref<5x8xf64>
+// CHECK-NEXT:            %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:            %b1 = memref.load %B[%{{.*}}, %{{.*}}] : memref<5x8xf64>
+// CHECK-NEXT:            %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:            %b2 = memref.load %B[%{{.*}}, %{{.*}}] : memref<5x8xf64>
+// CHECK-NEXT:            %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:            %b3 = memref.load %B[%{{.*}}, %{{.*}}] : memref<5x8xf64>
+// CHECK-NEXT:            %prod0 = arith.mulf %a0, %b0 fastmath<fast> : f64
+// CHECK-NEXT:            %prod1 = arith.mulf %a1, %b1 fastmath<fast> : f64
+// CHECK-NEXT:            %prod2 = arith.mulf %a2, %b2 fastmath<fast> : f64
+// CHECK-NEXT:            %prod3 = arith.mulf %a3, %b3 fastmath<fast> : f64
+// CHECK-NEXT:            %res0 = arith.addf %prod0, %c0 fastmath<fast> : f64
+// CHECK-NEXT:            %res1 = arith.addf %prod1, %c1 fastmath<fast> : f64
+// CHECK-NEXT:            %res2 = arith.addf %prod2, %c2 fastmath<fast> : f64
+// CHECK-NEXT:            %res3 = arith.addf %prod3, %c3 fastmath<fast> : f64
+// CHECK-NEXT:            scf.yield %res0, %res1, %res2, %res3 : f64, f64, f64, f64
+// CHECK-NEXT:          }
+// CHECK-NEXT:          %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:          memref.store %{{.*}}, %C[%{{.*}}, %{{.*}}] : memref<3x8xf64>
+// CHECK-NEXT:          %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:          memref.store %{{.*}}, %C[%{{.*}}, %{{.*}}] : memref<3x8xf64>
+// CHECK-NEXT:          %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:          memref.store %{{.*}}, %C[%{{.*}}, %{{.*}}] : memref<3x8xf64>
+// CHECK-NEXT:          %{{.*}} = affine.apply affine_map<(d0, d1) -> (((d0 * 4) + d1))> (%{{.*}}, %{{.*}})
+// CHECK-NEXT:          memref.store %{{.*}}, %C[%{{.*}}, %{{.*}}] : memref<3x8xf64>
+// CHECK-NEXT:        }
+// CHECK-NEXT:      }
+// CHECK-NEXT:      func.return %C : memref<3x8xf64>
+// CHECK-NEXT:    }
```

Comment:
I know this is contrary to previous practice, but would it be okay to use SSA names instead of the `%{{.*}}` here? I think some subtelties like the iteration bounds and which loop corresponds to what are lost otherwise.